src/main/java/com/ita/if103java/ims/service/impl/AssociateServiceImpl.java

```diff
@@ -70,8 +74,8 @@ public AssociateDto update(UserDetailsImpl user, AssociateDto associateDto) {
     }
```

Comment:
Just mentioned that address is not updated when update associate.
Please add here update of the address and also mark this method as transactional